harvard-lts/Harvard-Geospatial-Library
# Passenger

**Moving to Passenger like we run Curiosity**
* * *

# What does this Pull Request do?
This moves us to running similarly to how we run Curiosity, which has been more stable and also this way we're uniform. We moved to phusion image latest (so newer ruby), and using the actual phusion up command like in curiosity. This also bumps us to Rails 7 and the latest GeoBlacklight

# How should this be tested?

You can build locally to just make sure things come up, but this is up on dev to allow for full testing, 2 items we know are actually functioning in dev:

https://hgl-dev.lib.harvard.edu/catalog/harvard-ams7810-s250-u54-ng50-13
[10:07](https://harvard-huit.slack.com/archives/D01Q46LCEDV/p1678889243219319)
https://hgl-dev.lib.harvard.edu/catalog/harvard-meapophist

So you should be able to search, browse, and then view these 2 (others don't work because they aren't actually in -- junk records). Downloads don't work in dev but do in QA (this is an issue the predates the image here)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
@captmiddy -- I'm specifically interested in removing any bloat you see in the Dockerfile.
